build.gradle

```diff
@@ -37,6 +37,8 @@ dependencies {
 	runtimeOnly group: 'io.jsonwebtoken', name: 'jjwt-jackson', version: '0.12.6'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation group: 'org.springdoc', name: 'springdoc-openapi-starter-webmvc-ui', version: '2.5.0'
+	implementation 'me.paulschwarz:spring-dotenv:4.0.0'
```

Comment:
Seems that you are adding .env file. If this is the case you should also add `.env.example` that is NOT in `.gitignore` and is pushed to github. This file will contain all keys with empty or dummy values.
Here is a short explanation even though it is for different framework, the concept is still the same https://stackoverflow.com/questions/52084447/what-is-the-need-for-env-and-env-example-files-in-laravel